### ISSUE TITLE
Physiology descriptions in NCI taxa

### ISSUE THREAD

**Issue by kbeckenrode** [waldronlab/MicrobiomeOntology#4]
@lwaldron @lgeistlinger  @wdduncan 

I'm currently adding the NCITaxon into our working microbiome ontology, so I can start adding physiological terms (like Gram stain, oxygen utilization, size, etc)--but I just noticed that the NCITaxon have a detailed description at the phylum and species level. The NCITaxon adds some physiological details in the taxon description, as opposed to adding it as a property of the taxon.

Here is an example: [https://www.ebi.ac.uk/ols/ontologies/ncit/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FNCIT_C86398&viewMode=All&siblings=false](url)

Even though I want to add more description than what is detailed in the NCITaxon Ontology description, is it redundant to add Gram stain, oxygen utilization, size, and etc, to taxon within a separate ontology?

**Comment by lgeistlinger** [comment]
I think these summaries are great to parse these attributes for specific microbes, ie when populating the ontology by annotating entities (eg _Fusobacterium gonidiaformans_) to specific terms (eg _anaerobic_).

As I see it, these attributes would still need to be generally arranged in an ontology, without necessarily having specific microbes in mind, where I would suggest to maybe start with what the top levels of the ontologies might be, going from general to more specific, eg:

- microbial attribute (root)
-- oxygen utilization
--- aerobic
--- anaerobic
--- facultative anaerobic
-- gram stain
--- positive
--- negative
...

**Comment by kbeckenrode** [comment]
Thanks for your quick reply to ease my concerns. I just got really worried. Thanks for sketching out the top levels. I was also thinking something quite similar. 

I'm still not convinced about the hierarchy between the taxon and the attributes. I'm thinking that the "microbial attribute", as you have above, along with all the other nested terms will be linked below to each taxon. It's redundant, but I'm not sure how else do structure it. @lgeistlinger

**Comment by wdduncan** [comment]
I would start by extending PATO. Have you had a look at PATO's shape class?  
http://www.ontobee.org/ontology/PATO?iri=http://purl.obolibrary.org/obo/PATO_0000052 

PATO has a physical object quality class:  
http://www.ontobee.org/ontology/PATO?iri=http://purl.obolibrary.org/obo/PATO_0001241

There are also some classes about gram stains in the Adverse Event ontology:  
http://www.ontobee.org/ontology/OAE?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FOAE_0002444

**Comment by kbeckenrode** [comment]
@wdduncan  The PATO 'physical object quality' class is certainly looks promising for describing morphologies!

In terms of structuring annotations of taxons, what method do you think is best:

1. Taxons as the parent to all physiologies attributes, or
2. List all physiologies as classes and have taxons as children to different physiology classes?

**Comment by wdduncan** [comment]
Both views are useful, and you can have both. The best way to do this is to use defined classes to auto-classify the physical attributes. In Protege, defined classes have the triple-bar in the circle.

Suppose I was wanted to classify striated organisms. I would created a defined class something like this:

striated organism `equivalent to` organism and has quality some striated (see http://www.ontobee.org/ontology/PATO?iri=http://purl.obolibrary.org/obo/PATO_0001410). 

After I run the reasoner (ELK should be fine) all organisms that have the subclass axiom "has quality some striated" will show up as subclasses of the defined class in the inferred view of Protege. I don't think web Protege can do reasoning, but I may be wrong.

**Comment by kbeckenrode** [comment]
You're right. WebProtege is good for sketching out ideas, but it is limited. Maybe I'll have to go back to desktop version. 

I want to make sure I understand, so if I add the property of **striated organism**, I can define a class using the  **striated organism** property? 

Thanks Bill for the adding clarity

**Comment by wdduncan** [comment]
Hi Kelly. Please see the attached demo ontology (in the zip file). You will see that in the asserted ontology, the `striated_orgranism` class does not have any children. But after, I run reasoning, it does.

Before:
![image](https://user-images.githubusercontent.com/3186638/89933270-d02b2d00-dbdc-11ea-857a-c3f3eb00047f.png)

After reasoning:
![image](https://user-images.githubusercontent.com/3186638/89933335-e76a1a80-dbdc-11ea-8903-a8a520dfc658.png)

The work is done by the way I defined the `striated_organism` class:
![image](https://user-images.githubusercontent.com/3186638/89933428-0bc5f700-dbdd-11ea-98b1-663c6f16a829.png)

Hope this helps.

[striated-organism-demo.owl.zip](https://github.com/waldronlab/MicrobiomeOntology/files/5058680/striated-organism-demo.owl.zip)

**Comment by kbeckenrode** [comment]
This is really helpful! Thank you so much Bill!
